src/frontend/src/shared/hooks/use-set-cookie-auth.ts

```diff
@@ -0,0 +1,13 @@
+import { Cookies } from "react-cookie";
+
+const useSetCookieAuth = (tokenName: string, tokenValue: string) => {
+  const cookies = new Cookies();
```

Comment:
Move this outside of the function so we have just one instance of `Cookies` instead of creating one with each call